packages/expo-calendar/src/Calendar.ts

```diff
@@ -47,6 +47,7 @@ export type Event = {
   calendarId: string;
   title: string;
   location: string;
+  externalId?: string; // iOS
```

Comment:
```suggestion
  externalId?: string;
```
This is also part of the Android response, isn't it?

Reply:
Yeah, but I've not been able to make it work. Would need some help to finalise it on the Android side (the failing test is real :).